### ISSUE TITLE
Spotify create instead of just search

### ISSUE THREAD

**Issue by alleyhector** [alleyhector/astrosnax#24]
Originating from issue #13 I fixed the first issue, i.e. deciding which search terms to use, and actually ended up going with both since public playlists weren't really giving me a ton of great options searching based on either food items or transits. So instead I combined them so that the returned playlists are one of each. 

But I still think there might be a better overall way of actually _generating_ playlists based on foods, transits, moods. Of course I could also always manually add moods or other search terms (artists, genres) to each transit based on my own feelings about the transit, but even that seems a bit fraught, not to mention more work. 

So basically I'm still thinking through all this. 

This resource may help: https://www.postman.com/postman/postman-team-collections/request/zadn7c4/get-recommendations
